tests/backends/test_endianness.py

```diff
@@ -160,19 +161,25 @@ def test_backend_sample_endianness(
 
 
 @pytest.mark.parametrize(
-    "circ",
+    "init_state", [product_state("000"), product_state("010"), product_state("111")]
+)
+@pytest.mark.parametrize(
+    "block",
     [
-        QuantumCircuit(3, RX(0, torch.pi)),
-        QuantumCircuit(3, RX(1, torch.pi)),
-        QuantumCircuit(3, RX(2, torch.pi)),
+        RX(0, torch.pi),
+        RX(1, torch.pi),
+        RX(2, torch.pi),
     ],
 )
 def test_pulser_run_endianness(
-    circ: QuantumCircuit,
+    init_state: Tensor,
+    block: AbstractBlock,
 ) -> None:
+    register = Register.from_coordinates([(0.0, 10.0), (0.0, 20.0), (0.0, 30.0)])
```

Comment:
Just FYI, I changed this to a register of "far-away" qubits just so I could "ignore" the atomic interaction in the pulser backend. Otherwise, these tests would only pass for initial state 000, which is what was being done previously. That makes sense because in that scenario the atoms don't interact anyway.

Proper tests comparing pyqtorch with pulser backend in an interacting scenario will come with the work on emulated analog.